compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -2175,6 +2184,315 @@ LogicalResult OnlineAttentionOp::getResultTilePosition(
   return success();
 }
 
+static AffineMap getPartialResultMap(AffineMap map, AttentionOpDetail &opInfo) {
```

Comment:
nit/naming : getPartialReductionMap ?

Reply:
not really. it returns the map for a result, given its original result map.